### ISSUE TITLE
I found that even after following the tensorflow documentation for installing tf gpu, tensorflow and keras was still using CPU only for computation.

### ISSUE THREAD

**Issue by jidroid404** [tensorflow/tensorflow#22257]
Please go to Stack Overflow for help and support:

https://stackoverflow.com/questions/tagged/tensorflow

If you open a GitHub issue, here is our policy:

1. It must be a bug, a feature request, or a significant problem with documentation (for small docs fixes please send a PR instead).
2. The form below must be filled out.
3. It shouldn't be a TensorBoard issue. Those go [here](https://github.com/tensorflow/tensorboard/issues).

**Here's why we have that policy**: TensorFlow developers respond to issues. We want to focus on work that benefits the whole community, e.g., fixing bugs and adding features. Support only helps individuals. GitHub also notifies thousands of people when issues are filed. We want them to see you communicating an interesting problem, rather than being redirected to Stack Overflow.

------------------------

### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
- **Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device**:
- **TensorFlow installed from (source or binary)**:
- **TensorFlow version (use command below)**:
- **Python version**:
- **Bazel version (if compiling from source)**:
- **GCC/Compiler version (if compiling from source)**:
- **CUDA/cuDNN version**:
- **GPU model and memory**:
- **Exact command to reproduce**:

You can collect some of this information using our environment capture script:

https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh

You can obtain the TensorFlow version with

python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"

### Describe the problem
Describe the problem clearly here. Be sure to convey here why it's a bug in TensorFlow or a feature request.

### Source code / logs
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached. Try to provide a reproducible test case that is the bare minimum necessary to generate the problem.

**Comment by harshini-gadige** [comment]
@jidroid404  Hi, good that you resolved the issue. In order to look into updating the doc, could you please provide the following information.

Have I written custom code (as opposed to using a stock example script provided in TensorFlow):
OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device:
TensorFlow installed from (source or binary):
TensorFlow version (use command below):
Python version:
Bazel version (if compiling from source):
GCC/Compiler version (if compiling from source):
CUDA/cuDNN version:
GPU model and memory:
Exact command to reproduce:

**Comment by harshini-gadige** [comment]
#12388

**Comment by jidroid404** [comment]
System information 
1.I used a stock example script provided in TensorFlow
2. Windows 10  Home
3. Not on mobile device
3. Installed from binary
4. tf.__version__
'1.11.0-dev20180905'
5.python -V
Python 3.6.6 :: Anaconda, Inc.
6. Not Applicable 
7.Not Applicable
8. Cuda_9.0.176.2_windows
    CuDNN : 9.0 
9. Nvidia GPU used 940MX
    Memory :12 GB
10. pip install tf-nightly-gpu
       https://pypi.org/project/tf-nightly-gpu/


I had installed CUDA and cuDNN without breaking anything and also set the PATH variable correctly. Still when I was training a model using Keras GPU usage was zero while CPU was maxing out.
And evidently so
sess = tf.Session(config=tf.ConfigProto(log_device_placement=True))
showed no GPU0
Hope this was explainatory and suffices the cause.
